autogluon/utils/tabular/ml/trainer/model_presets/presets.py

```diff
@@ -87,6 +89,10 @@ def get_preset_models_classification(path, problem_type, objective_func, stoppin
             TabularNeuralNetModel(path=path, name='NeuralNetClassifier', problem_type=problem_type,
                                   objective_func=objective_func, stopping_metric=stopping_metric, hyperparameters=nn_options.copy()),
         )
+    if lr_options is not None:
```

Comment:
Is there a reason this wasn't added for regression as well?

Reply:
More generally, I didn't see `sklearn.linear_model.LinearRegression` anywhere?  Is this current PR only covering classification use-cases for now?

Reply:
One difficulty I think is sklearn offers little linear regressor functionality that is analogous to the args of its `LogisticRegression` class

Reply:
Good catch. Added Linear regression for REGRESSION problem type.